Makefile

```
@@ -27,26 +27,26 @@ help:
 build-push:
 	docker login ghcr.io
 	docker buildx create --name $(APP_ID) --driver docker-container --platform linux/amd64,linux/arm64/v8 --use || true
```

```suggestion
	docker buildx create --name $(APP_ID) --driver docker-container --platform linux/amd64,linux/arm64 --use || true
```
Oh, since you updated the architecture list in the workflow, this probably needs to be changed too.